elements/pf-accordion/docs/pf-accordion.md

```diff
@@ -90,7 +90,7 @@
   </pf-accordion>
   ```
 
-  ### Single Expanded Behavior
+  ### Single Expanded Behavior {class="anuj"}
```

Comment:
Maybe the solution here is to mock up the PatternFly components pages even closer by breaking these out into a h3 Examples and each being a subsection of `{% band header="Single Expanded Behavior", level="4" %}` instead of lumping them all under "Usage" as we currently are doing.  This would give you the background separation on each without having to directly modify `band.njk`.

```Suggestion
{% endband %}

Reply:
Thanks for the suggestions @zeroedin 
I have added div as a parent element for all the examples in the *.md file like this:
example-preview -> This CSS class will add white background color and padding as per upstream.
Please check the code for more details. Please let me know if you have suggestions.

<div class="example-preview">
<example code pf element render>
</div>